### DISCUSSION TITLE
Convert Eluer Orientation to Miller Orientation

### DISCUSSION THREAD

**Discussion by Mabroukimedcea** [mtex-toolbox/mtex#1875] [Ask Anything]
Hello to the Mtex community, 

I would like to know if it is possible to convert Eluer orientation to Miller orientation. Is there a function that I can use?

I am plotting the ODF (see figure below) and I want to obtain the location where the intensity is maximum in terms of the Miller orientation. Can I plot the ODF and obtain the Miller orientation from the ODF directly withou conversion? 
Thanks in advance 
Mohamed Mabrouki 

![image](https://github.com/mtex-toolbox/mtex/assets/126521440/1bafc074-166d-4be4-b2cc-9b2938ff342a)

**Comment by lfgmorales** [comment]
Hi

You may want to start here

https://mtex-toolbox.github.io/DefinitionAsCoordinateTransform.html

cheers

Luiz



On 19 Jun 2023, at 13:33, Mabroukimedcea ***@***.******@***.***>> wrote:



Hello to the Mtex community,

I would like to know if it is possible to convert Eluer orientation to Miller orientation. Is there a function that I can use?

I am plotting the ODF (see figure below) and I want to obtain the location where the intensity is maximum in terms of the Miller orientation. Can I plot the ODF and obtain the Miller orientation from the ODF directly withou conversion?
Thanks in advance
Mohamed Mabrouki

[image]<https://user-images.githubusercontent.com/126521440/246816753-1bafc074-166d-4be4-b2cc-9b2938ff342a.png>

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1875>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN43ZZKTM4SJJDFDXTLXMA2JZANCNFSM6AAAAAAZLZZO5Q>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1875", "url": "https://github.com/mtex-toolbox/mtex/discussions/1875", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by Mabroukimedcea** [reply]
thanks , but I still can't get how to convert 
I have my Eluer angel for exmple (   phi1  Phi phi2 = 25   75   45 )
and wanna the {hkl}<uvw> orientation 
If you have an exemple please or anyone else 
thanks in adavnced

**Reply by ralfHielscher** [reply]
{hkl} usually denotes a lattice plane. How can it determine an orientation?

**Reply by Mabroukimedcea** [reply]
I made a mistake when i was writing 
I want to get the orientation (texture) designed by {hkl} uvw. 
where {hkl} is the family of crystallographic plane, as you said 
and uvw is  the crystallographic direction

**Reply by Mabroukimedcea** [reply]
( when I wrote the direction uvw between<> it didn't appear )

Could you tell me how I can convert the Eluer angle that I get from my ODF into an orientation designed by {hkl} uvw and thank you very much for your help

**Reply by ralfHielscher** [reply]
you can the {hkl} and <uvw> by 
```matlab
hkl = round(ori \ zvector)
uvw = ori \ xvector;
uvw.dispStyle = 'uvw';
round(uvw)
```
I have some doubt that this is helpful though :)
Ralf.

**Reply by Mabroukimedcea** [reply]
Thanks you
I will try it and I let you know 
Mohamed Mabrouki
